components/DonateButton.tsx

```
@@ -1,70 +1,23 @@
-  Tooltip,
-  TooltipContent,
-  TooltipProvider,
-  TooltipTrigger,
-} from "../components/ui/tooltip";
+import Link from "next/link";
 
 interface CheckoutButtonProps {
   cartItems: { name: string; price: number; quantity: number }[];
 }
 
 const CheckoutButton: React.FC<CheckoutButtonProps> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)